src/en/holymanga/build.gradle

```
@@ -1,9 +1,9 @@
 ext {
     extName = 'HolyManga'
     extClass = '.HolyManga'
-    themePkg = 'zbulu'
+    themePkg = 'fmreader'
     baseUrl = 'https://w34.holymanga.net'
-    overrideVersionCode = 4
+    overrideVersionCode = 5
```

```suggestion
    overrideVersionCode = 2
```

since `baseVersionCode` changed from 6 to 9

---

fix it